### ISSUE TITLE
Help finding a sponsor for learning materials

### ISSUE THREAD

**Issue by juanlizarazo** [jasmine/jasmine#1636]
Hello! I know, I know, this is not an issue or a bug! It is just a question and I hope you can point me in the right direction here.

**TL;DR:** I created an awesome course (with the help of @reypena) on Jasmine that has paid students from 50+ countries and I want to make it available for free but I need a sponsor.

Hi! I am [Juan Lizarazo](https://twitter.com/juanlizarazog) 👋 , author of "[The unit testing your Javascript with Jasmine](https://www.udemy.com/unit-testing-your-javascript-with-jasmine/)" course at Udemy. 

![image](https://user-images.githubusercontent.com/5609353/49983108-e2435c80-ff1d-11e8-9724-97f07d3daded.png)

I wanted to create the best resource online where people could learn jasmine quick and get solid foundations but I never it expect it to become the highest rated course in Jasmine in the platform with students from 50+ countries. Nothing like spreading the jasmine love ❤️ ! 

So, I want to make the course **free, available for everyone, forever** so more people from developing countries can have access to it and can learn how to use the framework, but in order to do, I just need to recoup my production costs.

I need some guidance on whom I can contact to get a sponsor, I'd like to make the course free sooner rather than later as I could just wait 6 months to one year and then do so. 

Is this something pivotal labs or the jasmine project would be interested in doing?

**Comment by slackersoft** [comment]
Unfortunately, Pivotal doesn't really have a budget for Jasmine outside of a bit of time set aside for me, so I don't think we're likely to sponsor the course. Good luck!
